d2game/d2gamescreen/game.go

```diff
@@ -153,7 +153,7 @@ func (v *Game) Advance(tickTime float64) error {
 
 func (v *Game) bindGameControls() {
 	for _, player := range v.gameClient.Players {
-		if player.Id != v.gameClient.PlayerId {
```

Comment:
One more `Id` here.

Reply:
thank you!

Reply:
I'll put this in a PR for d2game lint errors